lib/stacks/elasticsearch-stack.ts

```diff
@@ -0,0 +1,130 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Do you propose we add such stacks to quick-start as opposed to patterns? 

It is surely a nice and handy stack. 

It is a more conceptual question: if customers are expected to use it as an example, e.g. clone and customize, then it should be in patterns. 

I am afraid it may be a too much for our team to incorporate quick starts for other managed services in here, we barely have resources for cluster level as it stands. Going down this path, we will have to provide stacks for AMP/AMG as well as RDS, Dynamo, etc. 

I see the value for the platform though, let's discuss.

Reply:
Sounds great. Let's discuss.